springwolf-core/src/main/java/io/github/stavshamir/springwolf/schemas/example/DefaultExampleXmlValueSerializer.java

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.stavshamir.springwolf.schemas.example;
+
+import org.w3c.dom.Document;
+
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import java.io.StringWriter;
+
+public class DefaultExampleXmlValueSerializer implements ExampleXmlValueSerializer {
+    @Override
+    public String writeDocumentAsXmlString(Document document) throws TransformerException {
+        DOMSource domSource = new DOMSource(document);
+        Transformer transformer = TransformerFactory.newInstance().newTransformer();
+        transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
+        transformer.setOutputProperty(OutputKeys.METHOD, "xml");
+        transformer.setOutputProperty(OutputKeys.ENCODING, "ISO-8859-1");
```

Comment:
Not sure which settings make sense here, I would probably change the encoding to utf-8 ...

Reply:
Can we guarantee the encoding?
In case it is not a required field, I would probably skip it